### PR TITLE
Reduce regression test run time

### PR DESCRIPTION
**Pull request by vanderhe**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,32 +12,30 @@ jobs:
...
+    defaults:
+      run:
+        shell: bash -l {0}
+
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        mpi: [nompi, openmpi]
+        mpi: [nompi, mpich]
```
Comment (bhourahine) [inline .github/workflows/build.yml:23]: What's the reason for changing framework?

**Review by bhourahine — APPROVED** [review]
Seems OK.

File: .github/workflows/build.yml (hunk 1 above)
Comment (vanderhe) [reply .github/workflows/build.yml:23]: Fetching `openmpi-mpifort` from conda-forge for whatever reason downgrades `gfortran` at the moment...

Comment (vanderhe) [reply .github/workflows/build.yml:23]: > Can you just comment it up then?

You mean adding a comment in the build.yml file?

Comment (bhourahine) [reply .github/workflows/build.yml:23]: Either the git log, or (and probably better) the yaml.

Comment (vanderhe) [reply .github/workflows/build.yml:25]: Comment added to the build.yml file.

**Pull request merged**